CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Change Log
 
+## [0.29.0](https://github.com/eclipse/lemminx/milestone/45?closed=1) (November 21, 2024)
+
+### Enhancements
+
+- ICompletionParticipant#onXMLContent is not called for CDATA. See [#1694](https://github.com/eclipse-lemminx/lemminx/issues/1694).
```

Comment:
I think this is a bug fix. i.e. we weren't considering completion inside CDATA as XML content, but now we are.

Reply:
move to bug